packages/products-used/style.module.css

```diff
@@ -0,0 +1,84 @@
+.root {
+  &.dark {
+    background-color: var(--wpl-neutral-900);
+  }
+}
+
+.eyebrow {
+  composes: g-type-label-strong from global;
+  margin: 0 0 var(--wpl-spacing-03) 0;
+  color: var(--wpl-neutral-500);
+
+  &.dark {
+    color: var(--wpl-neutral-400);
+  }
+}
+
+.productList {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 0;
+  list-style-type: none;
+  gap: var(--wpl-spacing-03);
```

Comment:
Looks like we have a consistent row-gap of 8px and only the column-gap changes from 8px to 16px.

Reply:
Ah, good catch. Updated.